src/state/queries/post-feed.ts

```diff
@@ -313,53 +313,22 @@ export function usePostFeedQuery(
                   const feedPostSlice: FeedPostSlice = {
```

Comment:
This is actually getting close to the shape of the "feed slice" class so I think we can unify these concepts in the future and maybe represent moderation as a feed tune.